frontend/src/metabase/dashboard/components/DashboardInfoSidebar.tsx

```diff
@@ -98,6 +99,7 @@ export function DashboardInfoSidebar({
           htmlFor={autoApplyFilterToggleId}
         >
           <Toggle
+            disabled={!canWrite}
```

Comment:
Prevent users without collection curate permission from toggling this toggle. I leave it visible like other settings. The toggle disable state isn't really distinguishable from the non-disabled state, so maybe we could use some design, but I do this for now so we can potentially ship this with the RC1 tomorrow.